DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Settings/DicomPerturbSetting.cs

```diff
@@ -0,0 +1,55 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Dicom.Anonymizer.Core.Exceptions;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+using Microsoft.Health.Dicom.DeID.SharedLib;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Settings
+{
+    public class DicomPerturbSetting : PerturbSetting, IDicomAnonymizationSetting
+    {
+        public PerturbDistribution Distribution { get; set; } = PerturbDistribution.Uniform;
+
+        public IDicomAnonymizationSetting CreateFromRuleSettings(string settings)
+        {
+            try
```

Comment:
possible to have a base class to do all these thing? most of these two functions for all the settings are quite similar